xdsl/dialects/builtin.py

```diff
@@ -1174,14 +1174,21 @@ class UnregisteredOpWithName(UnregisteredOp):
             @classmethod
             def create(
                 cls,
+                *,
```

Comment:
this seems like a great change to separate into its own PR, would make the changes here a little more focused